dynosaur_derive/src/lib.rs

```diff
@@ -56,9 +56,9 @@ pub fn dynosaur(
     .into()
 }
 
-fn mk_erased_trait(attrs: &Attrs, item: &ItemTrait) -> TokenStream {
+fn mk_erased_trait(item: &ItemTrait) -> TokenStream {
     let erased_trait = ItemTrait {
-        ident: attrs.ident.clone(),
+        ident: Ident::new(&format!("Erased{}", item.ident), item.ident.span()),
```

Comment:
Not entirely sure if using `MyTrait` span which is what `item.ident.span()` is, would be correct.

Reply:
I think I lean toward using `Span::call_site()`, because it would be confusing if error messages about this trait point to the original trait definition instead of the macro that defines it.